### PR TITLE
[RW-6723][risk=no] Split out WGS CDR publishing

### PR DESCRIPTION
**Pull request by calbach**
```
./project.rb publish-cdr ...
./project.rb publish-cdr-wgs ...
```

Only added config entries for WGS where we have WGS data.

### CODE REVIEW

File: api/db-cdr/generate-cdr/libproject/devstart.rb [hunk 1]
```diff
@@ -200,111 +275,104 @@ def publish_cdr(cmd_name, args)
...
 
-        if existing_groups.include?(new_group)
-          common.status "#{new_group} already in ACL, skipping..."
-        else
-          common.status "Adding #{new_group} as a READER..."
-          new_entry = { "groupByEmail" => new_group, "role" => "READER"}
-          json["access"].push(new_entry)
-        end
-      end
+def publish_cdr_wgs(cmd_name, args)
```
Comment (calbach) [inline api/db-cdr/generate-cdr/libproject/devstart.rb:316]: FYI: I've disabled this pattern, it was previously disabled in Codacy per https://docs.google.com/document/d/1o1nN4Oni9v6wgoYeFhG2yj47pqaD7XkVrRaTHqsdHhg/edit#heading=h.2f4f7hkj818a

File: api/db-cdr/generate-cdr/libproject/devstart.rb [hunk 2]
```diff
@@ -200,111 +275,103 @@ def publish_cdr(cmd_name, args)
...
-      existing_users = Set[]
-      for entry in json["access"]
-        if entry.key?("groupByEmail")
-          existing_groups.add(entry["groupByEmail"])
-        end
-        if entry.key?("userByEmail")
-          existing_users.add(entry["userByEmail"])
-        end
+  source_cdr_project = tier.fetch(:source_cdr_project)
+  dest_dataset = "#{tier.fetch(:dest_cdr_project)}:#{op.opts.bq_dataset}"
```
Comment (ericsong) [inline api/db-cdr/generate-cdr/libproject/devstart.rb:280]: nit: broad spec ops uses a pattern of prefixing `fq_` for datasets/tables that are fully qualified which I thought was pretty useful

**Review by ericsong — APPROVED** [review]
Looks good. much easier to understand than before, thanks for the changes.

File: api/db-cdr/generate-cdr/libproject/devstart.rb [hunk 3]
```diff
@@ -335,50 +396,48 @@ def create_wgs_extraction_dataset(cmd_name, args)
...
+  fq_datasets = [
+    "extractionCohortsDataset",
+    "extractionDestinationDataset",
+    "extractionTempTablesDataset"
+  ].map do |key|
+    fq_ds = extract_config.fetch(key)
+    raise ArgumentError.new("missing config value for #{key} in project #{op.opts.project}") unless fq_ds
+    raise ArgumentError.new("config value (#{fq_ds}) doesn't match expected CDR project for tier (#{cdr_project})") unless fq_ds.start_with? "#{cdr_project}."
+    fq_ds.sub("\.", ":")
+  end
```
Comment (calbach) [inline api/db-cdr/generate-cdr/libproject/devstart.rb:419]: @ericsong PTAL, I started creating the tmp dataset and found it too tedious. Instead - I've refactored this command to instead just create/update all WGS datasets for a given environment, as specified in the config. So the steps would be:

1. add dataset names to the config
2. run `create-wgs-extraction-datasets`

I've verified this works in staging.

File: api/db-cdr/generate-cdr/libproject/devstart.rb (hunk 2 above)
Comment (calbach) [reply api/db-cdr/generate-cdr/libproject/devstart.rb]: Switched to this pattern

**Review by ericsong — APPROVED** [review]
LGTM

**Pull request merged**